docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -331,6 +359,94 @@ The example below defines the following table access policy:
       ]
     }
 
+.. _system-file-function-rules:
+
+Function rules
+^^^^^^^^^^^^^^
+
+These rules control the user's ability to execute SQL functions, including PTFs.
+
+Each function rule is composed of the following fields:
+
+* ``user`` (optional): ``regex`` to match against user name. Defaults to ``.*``.
```

Comment:
what kind of regex? A SQL regex or a Java regex? Or something else. Also .. regex is not a keyword or so .. just say regular expression